docs/tutorial/topics/custom-icons.rst

```diff
@@ -162,8 +162,8 @@ However, as we haven't provided the actual icon files, the install fails, and
 Briefcase falls back to a default value (the same "gray bee" icon).
 
 Let's provide some actual icons. Download :download:`this icons.zip bundle
-<../resources/icons.zip>`, and unpack it into the root of your project
-directory. After unpacking, your project directory should look something like::
+<../resources/icons.zip>`, and unzip the file. We can then place the ``icons``
+folder into the root of the ``helloworld`` directory. Your project directory should look something like::
```

Comment:
So - the problem I've got with this rewording is that starts with talking about an `icons` zip file, and then talks about an `icons` folder - and it isn't clear where that came from.

If we're not going to add the full explicit instructions for how to unzip the file, my preference would be to just replace the potentially ambiguous "unpack" with "unzip, and leave it at that:
```suggestion
<../resources/icons.zip>`, and unzip it into the root of your project
directory. After unzipping, your project directory should look something like::
```

(as an aside - we wrap at an 88 character maximum).